src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.util.Scanner;
```

Comment:
Maybe this unused import should be deleted, I noticed the same issue in other places too.